webrtc.html

```diff
@@ -5826,8 +5826,10 @@ <h2>Methods</h2>
                     <li>
                       <p>Determine if negotiation is needed to transmit
                       <var>withTrack</var> in place of the sender's existing
-                      track. Negotiation is not needed if <var>withTrack</var>
-                      is null. Ignore which <code>MediaStream</code> the track
+                      track. Negotiation is not needed if no changes are
+                      required to the currently negotiated SDP offer/answer
+                      envelope, or <var>withTrack</var> is null.
```

Comment:
Something that just came up, in https://github.com/w3c/webrtc-pc/issues/1728: what if the current negotiated direction is "inactive"? Changes *are* required to send the track, technically. But I don't think we want to force the application to wait until the direction changes before calling `replaceTrack`. Do you agree? If so, do we make a specific exception for the direction, or is there some other way this could be worded...